elements/pf-alert/pf-alert.ts

```diff
@@ -0,0 +1,151 @@
+  danger: { viewBox: '0 0 512 512', path: 'M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zm-248 50c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z' },
+  info: { viewBox: '0 0 512 512', path: 'M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z' },
+  close: { viewBox: '0 0 352 512', path: 'M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z' },
+  get(name: 'close' | 'custom' | 'success' | 'warning' | 'danger' | 'info') {
+    const { viewBox, path } = ICONS[name];
+    return svg`
+      <svg
+          aria-hidden="true"
+          fill="currentColor"
+          height="1em" width="1em"
+          style="vertical-align:-0.125em"
+          viewBox="${viewBox}">
+        <path d="${path}"/>
+      </svg>`;
+  }
+};
+
+export class AlertCloseEvent extends ComposedEvent {
+  constructor() {
+    super('close', {
+      cancelable: true
+    });
+  }
+}
+
+const parseTimeout = (value: string): number => {
+  const parsed = Number.parseInt(value);
+  if (Number.isNaN(parsed)) {
+    return 8000;
+  } else {
+    return Number.parseInt(value);
+  }
+};
+
+/**
+ * Alert
+ * @slot - Place element content here
+ */
+@customElement('pf-alert')
+export class PfAlert extends LitElement {
+  static readonly styles = [styles];
+
+  @property({ reflect: true }) title = '';
+
+  /**
+  * The amount of time, in milliseconds, the alert will be visible before automatically closing.
+  * If set to `undefined`, the alert will not automatically close.
+  * If set to `true`, the alert will close after 8 seconds.
+  * */
+  @property({ reflect: true }) timeout: string | undefined = undefined;
```

Comment:
```suggestion
  @property({ converter: {/**/} }) timeout: boolean | number = false;
```

you'll need to write a converter, and typeguards at all the call sites